metomi/rose/opt_parse.py

```diff
@@ -29,22 +86,32 @@ class RoseOptionParser(OptionParser):
 
     """
 
+    DEFAULT_OPTS = {
+        "debug_mode",
+        "profile_mode",
+        "quietness",
+        "verbosity",
+    }
+
     OPTIONS = {
         "address_mode": [
             ["--address-mode", "--url", "-A", "-U"],
             {
                 "action": "store_const",
                 "const": "address",
                 "dest": "lookup_mode",
-                "help": "Shorthand for --lookup-mode=url",
+                "help": "Shorthand for --lookup-mode=address",
             },
         ],
         "all_revs": [
             ["--all-revs"],
             {
                 "action": "store_true",
                 "default": False,
-                "help": "Return all revisions of matched items.",
+                "help": (
+                    "Specify whether to search deleted suites and superceded"
```

Comment:
```suggestion
                    "Specify whether to search deleted suites and superseded"
```
https://www.collinsdictionary.com/dictionary/english/supercede